src/wmtk/utils/triangle_areas.hpp

```diff
@@ -1,62 +1,78 @@
 #pragma once
-#include <igl/predicates/predicates.h>
+
 #include <Eigen/Core>
+
+
 namespace wmtk {
-    class Tuple;
-    class TriMesh;
-    namespace attribute {
-    template <typename T>
-    class MeshAttributeHandle;
-    }
+class Tuple;
+class TriMesh;
+namespace attribute {
+template <typename T>
+class MeshAttributeHandle;
 }
+} // namespace wmtk
 namespace wmtk::utils {
-
-// template get 3d tri area
-template <typename ADerived, typename BDerived, typename CDerived>
-auto triangle_3d_area(
-    const Eigen::MatrixBase<ADerived>& a,
-    const Eigen::MatrixBase<BDerived>& b,
-    const Eigen::MatrixBase<CDerived>& c) -> typename ADerived::Scalar
+class Area
```

Comment:
Let's discuss that but I would prefer namespaces over classes. Advantage of namespaces:
1. You can split it up in multiple files.
2. You can use `using` which often makes sense if not used globally.
3. Namespaces can represent your folder/code structure which makes them more intuitive.
4. A `class` is a recipe for objects. A fully static class does not fulfill that purpose. In your code here, you can create an object of type `Area` which does not make any sense.

Additionally, the file and it's content should match. If you want to have a `class Area`, then the file should be called `Area.hpp`.